### ISSUE TITLE
Stuck at loading TaskBoard

### ISSUE THREAD

**Issue by Sabari-SGK** [kiswa/TaskBoard#607]
i am new to TaskBoard

It is stuck at loading taskboard

and ./build composer is not exists in the 1.o.x version

anyone please help me with this issue.

![image](https://user-images.githubusercontent.com/99186810/152791664-d2c77e3e-e16a-4a98-a1db-1f908352c8d3.png)

**Comment by marner2** [comment]
Can you post up any client side javascript errors that you are getting?

**Comment by Sabari-SGK** [comment]
![image](https://user-images.githubusercontent.com/99186810/154207420-22b56aa3-3b3a-442a-9b5a-1016469cc849.png)

Here is the image

**Comment by marcozucchi** [comment]
@Sabari-SGK @marner2 I get a similar issue, I guess the problem arises when the application doesn't find the files to render the Kanban board. 

For me, the issue was that the js was trying to access files at a wrong directory level. Changing the var path in the index.html file from 

`var = window.location.pathname`

to 

`window.location.pathname + '/../'`

This is not an ideal solution, but it works, at least for Linux devices

**Comment by marner2** [comment]
Yes the issue has to do with improper `<base>` tag settings. It's trying to automatically detect if it's in a subdirectory, but this doesn't always work.

The simplest thing to do is to manually put your own `<base>` tag into index.html and delete the javascript that is manipulating it there. A harder solution is to figure out exactly how you need to change the logic to make it work for your solution, and then post it here so we can handle your case.

**Comment by zajerao** [comment]
I had the same issue. Tried to install in a cpanel shared hosting with WHM access. Tried both in subdir and public_html.
Pasted the error from api error log in GPT and it said
/public_html/api/vendor/gabordemooij/redbean/RedBeanPHP/OODBBean.php
The syntax error appears to be due to the double backslashes (\\) before the interface names. In PHP, you don't escape backslashes in this context. It should likely be:
`class OODBBean implements \IteratorAggregate, \ArrayAccess, \Countable, Jsonable
`
And so I did and then I could log in. However there are many other bugs and errors after that making it hard to continue. Nevertheless, the robot fixed this one.
